WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -460,8 +460,55 @@ + (XCUICoordinate *)gestureCoordinateWithCoordinate:(CGPoint)coordinate applicat
   if (shouldApplyOrientationWorkaround) {
     point = FBInvertPointForApplication(coordinate, application.frame.size, application.interfaceOrientation);
   }
-  XCUICoordinate *appCoordinate = [[XCUICoordinate alloc] initWithElement:application normalizedOffset:CGVectorMake(0, 0)];
-  return [[XCUICoordinate alloc] initWithCoordinate:appCoordinate pointsOffset:CGVectorMake(point.x, point.y)];
+  XCUIElement *element = application;
+  if (isSDKVersionGreaterThanOrEqualTo(@"11.0")) {
```

Comment:
method signature should be changed if `shouldApplyOrientationWorkaround ` parameter is not used anymore

Reply:
For backwards compatibility reason, I think ```shouldApplyOrientationWorkaround``` is useful if SDK < 11.